src/Shared/ResourceUtilities.cs

```diff
@@ -190,14 +190,27 @@ internal static string FormatResourceString(string resourceName)
         /// <param name="resourceName">Resource string to load.</param>
         /// <param name="args">Optional arguments for formatting the resource string.</param>
         /// <returns>The formatted resource string.</returns>
-        internal static string FormatResourceString(string resourceName, params object[] args)
+        internal static string FormatResourceStringStripCodeAndKeyWord(string resourceName, params object[] args)
```

Comment:
Can you do this rename in a separate commit with a comment that it's a pure rename with no semantic change? This looked scary but actually wasn't!